org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ZipActionImpl.java

```diff
@@ -418,6 +433,25 @@ private void applyZipStream(
 		}
 	}
 
+	private void printZipActionDuration(final String inputName) {
+		if (getLogger().isInfoEnabled()) {
```

Comment:
ActionImpl reports this for DEBUG level logging and we should be consistent here.

Reply:
Yes I know, but that is exactly the purpose of the issue #462 and this PR.
Debug mode produces too much such as it's not usable and slows down the whole transformation process.